web/public/data/params.json

```diff
@@ -1,4 +1,4 @@
 {
   "max_date": "2024-09-23",
   "min_date": "2020-04-27"
-}
\ No newline at end of file
+}
```

Comment:
Let's revert this? Posix systems kinda expect a tailing newline
 https://www.baeldung.com/linux/files-end-with-newlines